src/ts/cmd/storage-adapter-mongodb/src/services/mongodb-client/operations/SubmodelRepositoryService.ts

```diff
@@ -35,19 +35,49 @@ class SubmodelRepositoryService {
 */
   async getSubmodels(): Promise<Submodel[]> {
     await this.dbClient.connect();
+
+
     return (await this.dbClient.getAll())
       .filter((x) => x.serializedSubmodel)
       .map((x) => JSON.parse(x.serializedSubmodel));
   }
 
+
+  async getSubmodel(submodelid:string): Promise<string> {
+    await this.dbClient.connect();
+
+    let equipmentDescription = md5(submodelid)
+    logger.debug("submodelId: "+equipmentDescription);
```

Comment:
this is not the submodelId. This is the md5 hash of the id. This log may confuse people

Reply:
right, updated